backend/controllers/appointmentDetails.js

```diff
@@ -43,34 +43,50 @@ module.exports = ({ prisonApi, whereaboutsApi }) => async (req, res) => {
     prepostData['post-court hearing briefing'] = createLocationAndTimeString(videoLinkBooking.post)
   }
 
+  const additionalDetails = {
+    ...(videoLinkBooking && { courtLocation: videoLinkBooking.main.court }),
+    comments: appointment.comment || 'Not entered',
+    // addedBy: formatName(staffDetails.firstName, staffDetails.lastName),
```

Comment:
This is okay to go back in now we fall back to prison api, maybe one for later though.

Reply:
John did that - I moved the code. Adding this back in is a bit more involved now we have 2 njks displaying this data, but we will do that next